packages/components/src/__future__/Tag/_docs/RemoveableTag.stories.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { v4 as uuidv4 } from "uuid"
+import { AcademyIcon, ActionOffIcon, AddIcon, TagIcon } from "~components/Icon"
+import { RemoveableTag } from "../RemoveableTag"
+import { TagColorKeys } from "../types"
+
+const meta = {
+  title: "Components/RemoveableTag",
+  component: RemoveableTag,
+  args: {
+    children: "My tag",
+    removeButtonProps: {
+      onClick: () => alert("Clicked"),
+      "aria-label": "remove tag icon",
+    },
+  },
+} satisfies Meta<typeof RemoveableTag>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
```

Comment:
The Playground should be the simplest form of the component, where the viewer can use Controls to see how it affects the component.

Not really sure what this story is doing (haven't looked at SB yet), but if it's showcasing something, then feel free to make it another story.

Reply:
Overall it looks like it's showcasing how to use the `onClick` method. I'd simplify this playground story, and maybe replace the `onClick` with a simple alert

Reply:
I've had previous PRs knocked back for demonstrating props in a way that we would never expect users to behave, so the Playground I've built showcases adding and removing tags, which is their intended use case.

I'll change the story to the `alert` example, but it would be good if we could decide on these principles in a document somewhere.

Reply:
Good call, I'll write something up.

Reply:
Cheers - but not trying to dump that in your lap. I think we were planning on writing up some code-standards as a team which will be great. But I'm just not sure when these things change, or who changes them - did it come up in a meeting or something?

Reply:
I don't think anything's changed, it might just be a misunderstanding around the application of what is meant by avoiding "demonstrating props in a way that we would never expect users to behave" -- which is why I'll write this up to discuss further/clarify.